routes/DeveloperApplication.ts

```diff
@@ -10,15 +10,21 @@ import OktaService from '../services/OktaService';
 import GovDeliveryService from '../services/GovDeliveryService';
 import SlackService from '../services/SlackService';
 import DynamoService from '../services/DynamoService';
-import { validateApiList } from '../util/validators';
+import { vaEmailValidator, validateApiList } from '../util/validators';
 import { DevPortalError } from '../models/DevPortalError';
+import { DeveloperApplicationRequestBody } from '../types';
 
 export const applySchema = Joi.object()
   .keys({
     apis: Joi.custom(validateApiList).required(),
     description: Joi.string().allow(''),
     email: Joi.string().email().required(),
     firstName: Joi.string().required(),
+    internalApiInfo: Joi.object().keys({
+      programName: Joi.string().required(),
+      sponsorEmail: Joi.string().email().required(),
```

Comment:
Does sponsorEmail not need to be a `va.gov` email?

Reply:
I don't think so, but I'm not sure tbh

Reply:
Confirmed that `va.gov` is required on sponsorEmail. Added the relevant validation and tests.